navigator-menu/eslint.config.cjs

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
Config is not complete:
`env` is missing, so it's complaining about `window` being global. Also es6 env as well.
`source: module` and `ecmaVersion: latest` should go in `parserOptions`, otherwise it complains about `import` and `export` being reserved keywords.
React version detection is not present, also import/resolver plugin is not set up.